src/hotspot/os_cpu/linux_arm/os_linux_arm.cpp

```diff
@@ -293,10 +294,10 @@ extern "C" int JVM_handle_linux_signal(int sig, siginfo_t* info,
 
   JavaThread* thread = NULL;
   VMThread* vmthread = NULL;
-  if (os::Linux::signal_handlers_are_installed) {
+  if (PosixSignals::are_signal_handlers_installed()) {
     if (t != NULL ){
       if(t->is_Java_thread()) {
-        thread = t->as_Java_thread();
+        thread = (JavaThread*)t;
```

Comment:
Mis-merge? Please put this back to t->as_Java_thread()

Reply:
Merge issue, fixed.